### DISCUSSION TITLE
About /config and git

### DISCUSSION THREAD

**Discussion by mckoda09** [itzg/docker-minecraft-server#3191] [Q&A] [answered]
Why there is no /config directory container like in mc-proxy?
Are there any examples of servers, that are developed and bundled with this?

**Comment by itzg** [comment, marked as answer]
I'm no sure what you mean since there is support for /config

https://docker-minecraft-server.readthedocs.io/en/latest/mods-and-plugins/#optional-plugins-mods-and-config-attach-points

**Reply by mckoda09** [reply]
thanks! exactly what i need. so i can bind my local config directory and it will be synchronized? (i need this to sync modified configs from my git repo to the container)
